web/src/contexts/ConfigContext.tsx

```diff
@@ -1,39 +1,24 @@
-import React, { createContext, useContext, useState, useEffect } from "react";
+import React, { createContext, useContext, useState, useEffect } from 'react';
 
 export interface ClusterConfig {
-  clusterName: string;
-  namespace: string;
   storageClass: string;
-  domain: string;
-  useHttps: boolean;
-  adminUsername: string;
-  adminPassword: string;
-  adminEmail: string;
   adminConsolePort?: number;
   localArtifactMirrorPort?: number;
-  databaseType: "internal" | "external";
   dataDirectory: string;
   useProxy: boolean;
   httpProxy?: string;
   httpsProxy?: string;
   noProxy?: string;
   networkInterface?: string;
   globalCidr?: string;
-  databaseConfig?: {
-    host: string;
-    port: number;
-    username: string;
-    password: string;
-    database: string;
-  };
 }
 
 interface PrototypeSettings {
   skipValidation: boolean;
   failPreflights: boolean;
   failInstallation: boolean;
   failHostPreflights: boolean;
-  clusterMode: "existing" | "embedded";
+  clusterMode: 'existing' | 'embedded';
```

Comment:
Seems like we're struggling with formatting/ style options. Thoughts on setting up - https://www.npmjs.com/package/prettier ?

Reply:
yes, we have a bad history with prettier 😅: https://github.com/replicatedhq/kots/pull/5260

Reply:
Well, honestly I don't really care about what we use, as long as we have something setup in the project we can all rely on without the need to bikeshed too much around it.

Reply:
yeah, we should look into it